.github/scripts/review_agent.py

```
@@ -30,22 +30,28 @@
 
 
 def get_pr_diff(pr_num: str) -> str:
-    result = subprocess.run(
-        f"gh pr diff {pr_num} --patch",
+    try:
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)